### PR TITLE
Recognize protein complexes and proteins

### PR DESCRIPTION
**Pull request by egonw**

### CODE REVIEW

**Comment by egonw** [comment]
Closely related to https://github.com/WDscholia/scholia/pull/1543

File: scholia/app/templates/base.html [hunk 1]
```diff
@@ -395,17 +395,23 @@
...
+    UNION
     { wd:{{ q }} wdt:P50 | wdt:P2093 [] . BIND("work" AS ?aspect) }
     }
     `;
      settings = {
 	 headers: { Accept: 'application/sparql-results+json' },
 	 data: { query: query }
      };
-     
+     console.log(query)
```
Comment (fnielsen) [inline scholia/app/templates/base.html:409]: Maybe you want to remove the console log?

File: scholia/app/templates/base.html [hunk 2]
```diff
@@ -395,17 +395,23 @@
     UNION
     { wd:{{ q }} wdt:P644 [] . BIND("gene" AS ?aspect) }
     UNION
+    { wd:{{ q }} p:P31/ps:P31 wd:Q22325163 . BIND("complex" AS ?aspect) }
```
Comment (fnielsen) [inline scholia/app/templates/base.html:398]: Why not wdt:P31?

**Comment by fnielsen** [comment]
There were some extra commits here by me:

06c3b87aaed482fe7e1268786021b90340e0b2c2: Change path query to direct query

c9f50949ba507b9541dcd0ded600be3548ec0bf2: Made the complex type guesser broader

File: scholia/app/templates/base.html (hunk 2 above)
Comment (egonw) [reply scholia/app/templates/base.html:398]: Because when there is more than one statement with that property, then the RDF becomes less predictable.

Comment (fnielsen) [reply scholia/app/templates/base.html:398]: I do not understand. Would multiple statements not resolve in multiple wdt's and muliple (p/ps)'s?

**Pull request merged**